README.md

```diff
@@ -1,55 +1,86 @@
+<!---Table of commands with columns: 'command' and 'description'
+--->
 
-```
+| Command | Description |
+| --- | --- |
+| Add Connection | Add a Hyperledger Fabric instance connection |
+| Add Identity To Connection | Add an identity to be used when connecting to a Hyperledger Fabric instance  |
+| Connect To Blockchain | Connect to a Hyperledger Fabric blockchain using a blockchain connection |
+| Create Smart Contract Project | Create a new JavaScript or TypeScript smart contract project |
+| Delete Connection | Delete a Hyperledger Fabric instance connection |
+| Delete Package | Delete a smart contract package |
+| Disconnect From Blockchain | Disconnect from the blockchain you're currently connected to |
+| Install Smart Contract | Install a smart contract package onto a peer |
+| Instantiate Smart Contract | Instantiate an installed smart contract package onto a channel<br>**Note: This currently doesn't work with IBM Blockchain Platform Starter Plan and Enterprise plan - Coming soon!*|
+| Package a Smart Contract Project | Create a new smart contract package from a project in the Explorer|
+| Refresh Blockchain Connections | Refresh the Blockchain Connections view |
+| Refresh Smart Contract Packages | Refresh the Smart Contract Packages view |
+| Restart Fabric Runtime | Restart a Hyperledger Fabric instance |
+| Start Fabric Runtime | Start a Hyperledger Fabric instance |
+| Toggle Development Mode | Toggle the Hyperledger Fabric instance development mode |
 
+## Smart Contract Development Lifecycle
 <!---Things you can do in the Explorer view once the extension is installed
 --->
-## Start a new Fabric chaincode project
+### Create a new Fabric smart contract project
 <!---Short explanation with code-blocks
 --->
-![Start a new Fabric chaincode project](https://github.com/simran-sohanpal/blockchain-vscode-extension/blob/readmeupdates/client/media/smart_Contract_project-2.gif "Start a new Fabric chaincode project")
+
+<img src="https://github.com/IBM-Blockchain/blockchain-vscode-extension/blob/master/client/media/start_new_project_small.gif" width="85%" alt="Start a new Fabric smart contract project">
 <!---Link to docs with further instructions
 --->
-## Edit/ write chaincode files
+
+### Edit / write smart contract files
 <!---Short explanation with code-blocks
 --->
-![Edit chaincode files](https://github.com/simran-sohanpal/blockchain-vscode-extension/blob/readmeupdates/client/media/4%20.edit%20chaincode.png "Edit chaincode files")
+
+<img src="https://github.com/IBM-Blockchain/blockchain-vscode-extension/blob/master/client/media/edit_write_files.png" width="85%" alt="Edit smart contract files">
 <!---Link to docs with further instructions
 --->
-## Create a package from a chaincode project
-Coming in a later version. See [issue #2](https://github.ibm.com/IBM-Blockchain/fabric-vscode-extension/issues/2) 
+
+### Create a package from a smart contract project
```

Comment:
Change to "Package a smart contract project" or "Create a smart contract package".

I don't mind which.